map_gen/maps/crash_site/commands.lua

```diff
@@ -52,22 +52,41 @@ callback =
     end
 )
 
-local entities_to_check = {
+local static_entities_to_check = {
     'spitter-spawner','biter-spawner',
     'small-worm-turret', 'medium-worm-turret','big-worm-turret', 'behemoth-worm-turret',
-    'small-spitter', 'medium-spitter', 'big-spitter', 'behemoth-spitter',
-    'small-biter', 'medium-biter', 'big-biter', 'behemoth-biter',
     'gun-turret', 'laser-turret', 'artillery-turret', 'flamethrower-turret'
 }
+    
+local biter_entities_to_check = {
+    'small-spitter', 'medium-spitter', 'big-spitter', 'behemoth-spitter',
+    'small-biter', 'medium-biter', 'big-biter', 'behemoth-biter'
+}
 
-local function map_cleared()
+local function map_cleared(player)
+    player = player or server_player
     local get_entity_count = game.forces["enemy"].get_entity_count
-    for i = 1, #entities_to_check do
-        local name = entities_to_check[i]
+    -- Check how many of each turrets, worms and spawners are left and return false if there are any of each left.
+    for i = 1, #static_entities_to_check do
+        local name = static_entities_to_check[i]
         if get_entity_count(name) > 0 then
+            player.print('All enemy spawners, worms, buildings, biters and spitters must be cleared for non-admin restart.')
```

Comment:
```suggestion
            player.print('All enemy spawners, worms, buildings, biters and spitters must be cleared before Crashite can be restarted.')
```
For consistency with the other message?

Reply:
Agreed, updated.